src/main/java/com/timelogr/enterprise/TimeLogrController.java

```diff
@@ -5,51 +5,55 @@
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.RequestMapping;
 
-//ALL WEB FILES GO IN THE resources/templates DIRECTORY
+/**
+ * ALL WEB FILES GO IN THE resources/templates DIRECTORY
+ */
 @Controller
 @RequestMapping("/")
 public class TimeLogrController {
-    private TimeLogrServices timeLogrServices;
 
+    private final TimeLogrServices timeLogrServices;
 
     @Autowired
     public TimeLogrController(TimeLogrServices timeLogrServices){
         super();
         this.timeLogrServices = timeLogrServices;
     }
 
-
-
+    /**
+     * STARTS PROGRAM IN THE resources/templates/index.html
+     */
     @RequestMapping("/")
     public String index(Model model){
-        model.addAttribute("employee", this.timeLogrServices.GetAllEmployees());
-        return "index"; //STARTS PROGRAM IN THE resources/templates/index.html
+        model.addAttribute("employee", this.timeLogrServices.getAllEmployees());
+        return "index";
     }
 
     @RequestMapping("/clients")
-    public String Clients(Model model){
-        model.addAttribute("employee", this.timeLogrServices.GetAllEmployees());
+    public String clients(Model model){
```

Comment:
I changed your functions to follow proper Java naming conventions: https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html

Reply:
Yes, good.